apps/omg/test/omg/typed_data_hash_test.exs

```diff
@@ -18,8 +18,8 @@ defmodule OMG.TypedDataHashTest do
   * tier 1: acknowledged third party (Metamask) signatures we can verify (recover address from)
   * tier 2: final structural hash on prepared transaction that gives the same signatures as above
   * tier 3: intermediate results of hashing (domain separator, structural hashes of inputs & outputs)
-  * tier 4: end-to-end test of generating signatures in elixir code and verifying them in solidity library (comming
-            soon, TODO: ref :pr: here)
+  * tier 4: end-to-end test of generating signatures in elixir code and verifying them in solidity library (
+    done in https://github.com/omisego/elixir-omg/pull/656)
```

Comment:
if we want to leave a link here, link to code (test module name?) seems more helpful